File: assignment 10 - C# practice problems/10.cs
//Write a c# program to convert a given seconds to hours and minutes. 

using System;
namespace MyNameSpace
{
    class Program10
    {
        public static void SecondToHourAndMinute()
        {

            Console.WriteLine("Enter seconds: ");
            double secondInput = double.Parse(Console.ReadLine());

            secondInput /= 60;
            Console.WriteLine((int)(secondInput / 60) + " Hour/s " + (int)(secondInput % 60) + " Minute/s");

        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/10.js
//Write a JavaScript program to convert a given seconds to hours and minutes. 
function secondToHourAndMinute() {
    let numbersInput = prompt("Enter seconds");

    if (numbersInput != null) {
        numbersInput = numbersInput.trim();
        let num1 = parseFloat(numbersInput);
        num1 /= 60;
        alert(parseInt(num1 / 60) + " Hour/s " + parseInt(num1 % 60) + " Minute/s");
    }
}
secondToHourAndMinute()